staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1758,12 +1758,12 @@ const (
 	// CacheWriteBack - I/O from the guest is cached on the host.
 	CacheWriteBack DriverCache = "writeback"
 
-	// IOThreads - User mode based threads with a shared lock that perform I/O tasks. Can impact performance but offers
+	// DriveIOThreads - User mode based threads with a shared lock that perform I/O tasks. Can impact performance but offers
 	// more predictable behaviour. This method is also takes fewer CPU cycles to submit I/O requests.
-	IOThreads DriverIO = "threads"
-	// IONative - Kernel native I/O tasks (AIO) offer a better performance but can block the VM if the file is not fully
+	DriverIOThreads DriverIO = "threads"
+	// DriveIONative - Kernel native I/O tasks (AIO) offer a better performance but can block the VM if the file is not fully
 	// allocated so this method recommended only when the backing file/disk/etc is fully preallocated.
-	IONative DriverIO = "native"
+	DriverIONative DriverIO = "native"
```

Comment:
My understanding was that repeating the name of the type within the constant is an anti-pattern in golang and is to be avoided? See https://github.com/kubevirt/kubevirt/pull/11927/commits/1794b309c7b9194df534f000e315a381d9e1ff7d#diff-5e2591267ff7e9303f97ccac213986c2157131e3487a04a122b7662d401f7a57 where I'm proposing cleaning up some constant names (and their values but that's unrelated to your change) within the instance type API.

Reply:
I understand but in this case I still thing is better because otherwise, this option could be confused with the IOthread which is a different feature, see https://github.com/kubevirt/kubevirt/pull/12113

Reply:
or do you have a better name in mind?

Reply:
Just `Native` in this case?

Reply:
OK, the important aspect to me is that we don't use `IOThread`, so this can become `Thread`